pkg/cmd/admin/prune/images.go

```diff
@@ -27,9 +28,19 @@ import (
 )
 
 const (
-	imagesLongDesc = `%s %s - prunes images`
 	// PruneImagesRecommendedName is the recommended command name
 	PruneImagesRecommendedName = "images"
+
+	imagesLongDesc = `Prune images no longer needed by the system due to age and status.
```

Comment:
Yes, I let myself inspire by our [docs](https://docs.openshift.com/enterprise/3.0/admin_guide/pruning_resources.html#pruning-images) and didn't dare to fix it. Even though it seemed phony to me. Now I'm not alone, so I'm gonna fix it :-).